content/en/blog/_posts/2022-08-31-cgroupv2-ga.md

```diff
@@ -118,8 +118,8 @@ Scenarios in which you might need to update to cgroup v2 include the following:
   DaemonSet for monitoring pods and containers, update it to v0.43.0 or later.
 * If you deploy Java applications, prefer to use versions which fully support cgroup v2:
     * [OpenJDK / HotSpot](https://bugs.openjdk.org/browse/JDK-8230305): jdk8u372, 11.0.16, 15 and later
-    * [IBM Semeru Runtimes](https://www.eclipse.org/openj9/docs/version0.33/#control-groups-v2-support): jdk8u345-b01, 11.0.16.0, 17.0.4.0, 18.0.2.0 and later
-    * [IBM Java](https://www.ibm.com/docs/en/sdk-java-technology/8?topic=new-service-refresh-7#whatsnew_sr7__fp15): 8.0.7.15 and later
+    * [IBM Semeru Runtimes](https://www.ibm.com/support/pages/apar/IJ46681): 8.0.382.0, 11.0.20.0, 17.0.8.0, and later
```

Comment:
When we update existing blog articles, we should add a comment to the article noting that there's been a subsequent update.

Also, please check with the article authors that they don't object to this change.

Reply:
@sftim

> add a comment to the article noting that there's been a subsequent update

Do you have an example page that has such a notice so that I can copy the formatting?

Reply:
@bobbypage @sftim ping for previous questions